kbase-extension/scss/partials/_bulkImportCell.scss

```diff
@@ -98,15 +98,15 @@ $button-outdated: rgb(247, 155, 34);
           cursor: pointer;
         }
 
-      /* .kb-bulk-import__filetype_panel__filetype_button--selected */
-      &--selected {
+        /* .kb-bulk-import__filetype_panel__filetype_button--selected */
```

Comment:
I ran `stylelint` over the scss files, and it fixed some spacing and ordered stanzas according to its rules. There are no changes that result in different css output in this file.